File: bench/Reference/file_write_read_large.js
const fs = require('fs');
const data = fs.readFileSync("bench/data/large_text.txt");
fs.writeFileSync("/tmp/node_bench_large_write.txt", data);
const data2 = fs.readFileSync("/tmp/node_bench_large_write.txt");
console.log(data2.length);

File: bench/Reference/file_write_read_large.java
import java.nio.file.*;

public class file_write_read_large {
    public static void main(String[] args) throws Exception {
        byte[] data = Files.readAllBytes(Paths.get("bench/data/large_text.txt"));
        Files.write(Paths.get("/tmp/java_bench_large_write.txt"), data);
        byte[] data2 = Files.readAllBytes(Paths.get("/tmp/java_bench_large_write.txt"));
        System.out.println(data2.length);
    }
}
